gax-java/gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpRequestRunnableTest.java

```diff
@@ -64,6 +68,15 @@ public static void setUp() {
             .setTypeUrl("small")
             .build();
 
+    bodyRequestMessage =
```

Comment:
Since this message and the formatter below is only used by the single test, I would put it together with the test case so it's easier to read, see [here](go/unit-testing-practices#setUp) for the best practices.

Reply:
Sure will update

Reply:
This `bodyRequestMessage` could be moved to the unit test as well.

Reply:
Thanks! Moved the `bodyRequestMessage` field into the unit test.